constructor/preconda.py

```diff
@@ -15,7 +15,7 @@
 except:
     import ruamel_json as json
 
-files = 'urls', 'urls.txt', '.install.py'
+files = 'system.info', 'urls', 'urls.txt', '.install.py'
```

Comment:
So it appears this is going in the `pkgs/` package cache directory.  Is `system.info` the correct name for the file?  I'm not sure.  Which "system" are we talking about?  The system we're unpacking to, or the system that built the installer...   Would something like `constructor-build.info` be a better name?  Maybe better would be `.constructor-build.info`.